support/cas-server-support-throttle-core/src/main/java/org/apereo/cas/web/support/InMemoryThrottledSubmissionByIpAddressAndUsernameHandlerInterceptorAdapter.java

```diff
@@ -23,9 +24,10 @@ public InMemoryThrottledSubmissionByIpAddressAndUsernameHandlerInterceptorAdapte
                                                                                       final String usernameParameter,
                                                                                       final String authenticationFailureCode,
                                                                                       final AuditTrailExecutionPlan auditTrailExecutionPlan,
-                                                                                      final String applicationCode) {
+                                                                                      final String applicationCode,
+                                                                                      final ConcurrentMap map) {
         super(failureThreshold, failureRangeInSeconds, usernameParameter,
```

Comment:
Nicely done. Once lombok adds support for InheritedConstructors, all of this noise would too go away.